streams/src/main/java/org/apache/kafka/streams/processor/FailOnInvalidTimestamp.java

```diff
@@ -60,9 +63,14 @@ public long onInvalidTimestamp(final ConsumerRecord<Object, Object> record,
                                    final long recordTimestamp,
                                    final long previousTimestamp)
             throws StreamsException {
+        log.error("Input record {} has invalid (negative) timestamp. " +
```

Comment:
your message for the logger and for the exception, could you please extract it to `val`?